### ISSUE TITLE
Mos will not recognize that I granted Accessibility access

### ISSUE THREAD

**Issue by chuim** [Caldis/Mos#534]
- 问题描述 | Issue description:

I have been using Mos for a long time but a couple weeks ago it just stopped working. I relaunched it and it requested that I allowed Accessibility access to it from the System Preferences. As I had already been using it, that was already granted long ago. I tried denying access and regranting but Mos keeps on asking me to grant it no matter what.

- 如何重现 | How to reproduce it:

I'm not sure as this just started happening. Maybe it was tied to Mos being updated the last time I ran `brew update && brew upgrade`?

- 期望结果 | What is expected:

Mos should notice that the Accessibility permission was granted and start/work normally as before.

- 实际结果 | What is actually happening:

Mos is nor working and shows the "Needs access to Accessibility controls" message in the top bar menu. 

- 软件版本 | Application version:

I just manually updated it to the latest 3.4.1 and the problem is still there.

- 系统版本 | System version:

macOS Monterey 12.6

- 您系统中的其他鼠标管理软件 | Other mouse management application in your system:

None.

**Comment by daslicht** [comment]
have you tried to disable the permission and re-enable it ?

**Comment by tpeters67** [comment]
Same issue was working and then updated via brew yesterday. Have turned accessibility multiple times restart app and restarted machine it never seems to detect that the accessibility access has been granted.

**Comment by chrisonhub** [comment]
I saw the same issue - the answer is not to just "untick" the permission, but to select it and use the "-" button below to remove it.  Quit MOS, remove the permission and then launch MOS.

**Comment by greendog99** [comment]
> the answer is not to just "untick" the permission, but to select it and use the "-" button below to remove it. Quit MOS, remove the permission and then launch MOS.

Came here with the same issue. This fixed it. Thanks!

**Comment by tpeters67** [comment]
Yep thanks for the reminder I had the same issue PopClip a month or so ago but had forgotten this fixed it.

**Comment by amon-666** [comment]
same issue here

**Comment by chuim** [comment]
@chrisonhub's answer worked for my case too. Thanks!

I'll let the maintainers decide if this issue should be closed or not, depending on it being cause by an actual problem with Mos or being a bug in macOS.

**Comment by seff** [comment]
me too

**Comment by analogue-interface** [comment]
> I saw the same issue - the answer is not to just "untick" the permission, but to select it and use the "-" button below to remove it. Quit MOS, remove the permission and then launch MOS.

Worked for Me!!!

**Comment by Victorasa21** [comment]
> I saw the same issue - the answer is not to just "untick" the permission, but to select it and use the "-" button below to remove it. Quit MOS, remove the permission and then launch MOS.

worked for me, thanks!! :)
